electrum/wizard.py

```diff
@@ -287,6 +288,9 @@ def start(self, initial_data: dict = None) -> WizardViewState:
         self._current = WizardViewState(start_view, initial_data, params)
         return self._current
 
+    def is_xpub_password(self, wizard_data):
+        return wizard_data['xpub_password']
```

Comment:
shouldn't this return a bool instead of a str?
and is that key guaranteed to always be set? (well, I guess the way it is used *now*, it will be set)